src/doc/tutorials/CCL.rst

```diff
@@ -0,0 +1,266 @@
+.. _ccl:
+
+Connected Components
+=====================
+
+.. toctree::
+    :maxdepth: 2
+
+VisIt provides an expression and set of queries to help identify and summarize connected subcomponents of a mesh. 
+These capabilities can help isolate or compute statistics of complex features embedded in your data.
+The connected components algorithm used is unique in that it can not only process simple meshes, but it can also efficiently handle large meshes partitioned in a distributed-memory setting. This tutorial provides an introduction of how to use VisIt's connected components capabilities. The algorithm is not discussed here, for more details on the algorithm see [#ccl_bib_ref]_ .
```

Comment:
Change VisIt to VisIt_ throughout the tutorial.

Reply:
Changed most, I didn't change the ones that are possessive (VisIt's) not sure if we do that elsewhere?